css/standalone/dashboard.scss

```
@@ -1316,6 +1316,7 @@ $break_tablet: 1400px;
                 color: rgb(39, 39, 39);
                 font: 1em var(--fa-font-solid);
                 position: absolute;
+                font-family: 'Font Awesome 5 Free';
```

Maybe this fix to prevent usage of hardcoded font-family.
```suggestion
                font: var(--fa-font-solid);
                font-size: 1em;
                position: absolute;
```

---

There isa lot of `font: {$size} var(--fa-font-solid);` syntax in our scss files. I guess they all have to be fixed. I think the best way to do this is to use following syntax
```
font: var(--fa-font-solid);
font-size: {$size};
```